code/modules/core_implant/core_implant.dm

```diff
@@ -147,7 +147,7 @@
 
 
```

Comment:
Make a new function for this. Some other litany might have a 'uses all power' thing.